### PR TITLE
feat: mcq and result with score

### PR DESCRIPTION
**Pull request by sharmayatendra**
# Changes

- questions MCQ
- result page with score

image
image

### CODE REVIEW

**Review by vikrantkode — APPROVED** [review]
- Working fine as presented
- Code is cleanly written
- try to import files from single file i.e., `index.js` file
- good to merge

**Review by yashpatel1797 — APPROVED** [review]
looks good to merge.

File: src/App.js [hunk 1]
```diff
@@ -3,14 +3,18 @@ import Mockman from "mockman-js";
...
+import { Quiz } from "./pages/Quiz/Quiz";
+import { Result } from "./pages/Result/Result";
 
 function App() {
   return (
     <Routes>
       <Route path="/" element={<Homepage />}></Route>
       <Route path="/categories" element={<Categories />}></Route>
-      <Route path="/rules" element={<Rules />}></Route>
+      {/* <Route path="/rules" element={<Rules />}></Route> */}
```
Comment (yashpatel1797) [inline src/App.js]: remove commented code here

File: src/App.js [hunk 2]
```diff
@@ -3,14 +3,18 @@ import Mockman from "mockman-js";
 import { Route, Routes } from "react-router-dom";
 import { Homepage } from "./pages/Homepage/Homepage";
 import { Categories } from "./pages/Categories/Categories";
-import { Rules } from "./pages/Rules/Rules";
+// import { Rules } from "./pages/Rules/Rules";
```
Comment (yashpatel1797) [inline src/App.js]: remove here as well.

File: src/components/Question/Question.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import axios from 'axios'
+import "./Question.css"
+import { useEffect , useState } from 'react'
+import {useNavigate} from 'react-router-dom'
+import { useQuiz } from '../../context/quiz-context'
```
Comment (yashpatel1797) [inline src/components/Question/Question.jsx:5]: create `index.js` file to aviod doing `"../../"`

**Review by yashpatel1797 — APPROVED** [review]
code looks clean
file structure is good
design looks consistent

**Pull request merged**